tests/commonlib/kubernetes.py

```diff
@@ -179,7 +187,83 @@ def delete_resources(self, resource_type: str, **kwargs):
     def patch_resources(self, resource_type: str, **kwargs):
         """
         """
-        return self.dispatch_patch[resource_type](**kwargs)
```

Comment:
minimal docstrings should be provides.
We are working with PEP8 standards, you can find more information [PEP8 - Documentation strings](https://peps.python.org/pep-0008/#documentation-strings)